tests/test_lp.py

```diff
@@ -76,6 +76,26 @@ def test_linear(self):
         self.assertFalse(cqm.constraints)
         self.assertTrue(cqm.objective.is_equal(x + y))
 
+    def test_maximize(self):
+        lp = """
+        Maximize
```

Comment:
I see the current `reader` handles it, but just for end-to-end sanity check, perhaps add a test with invalid objective section keyword?